usecases/modules/modules.go

```diff
@@ -723,7 +723,7 @@ func (p *Provider) VectorFromSearchParam(ctx context.Context,
 		}
 	}
 
-	panic("VectorFromSearchParam was called without any known params present")
+	return nil, "", fmt.Errorf("could not vectorize input for collection %v with search-type %v and parameters %v. Make sure a vectorizer module is configured for this class", className, param, params)
```

Comment:
Makes sense. I propose `class` -> `collection`

Minor suggestion: 

Would it be worth surfacing the targetVector? e.g.: 
`fmt.Errorf("could not vectorize input for collection %v (target vector %v) with search-type %v and parameters %v. Make sure a vectorizer module is configured for this class", className, targetVector, param, params)`

Reply:
changed :)